builder/amazon/chroot/builder.go

```diff
@@ -48,11 +48,11 @@ type Config struct {
 	// in the Chroot Mounts section. Please read that section for more
 	// information on how to use this.
 	ChrootMounts [][]string `mapstructure:"chroot_mounts" required:"false"`
-	// How to run shell commands. This defaults to {{.Command}}. This may be
+	// How to run shell commands. This defaults to `{{.Command}}``. This may be
```

Comment:
Is the double back-tick ` here on purpose in this file ?
```suggestion
	// How to run shell commands. This defaults to `{{.Command}}``. This may be
```

Reply:
ah no that is a mistake, thanks for catching!